app/utils/env.server.ts

```diff
@@ -1,28 +1,34 @@
-import { invariant } from './misc.ts'
-
-const requiredServerEnvs = [
-	'NODE_ENV',
-	'DATABASE_PATH',
-	'DATABASE_URL',
-	'SESSION_SECRET',
-	'INTERNAL_COMMAND_TOKEN',
-	'CACHE_DATABASE_PATH',
-	// If you plan to use Resend, uncomment this line
-	// 'RESEND_API_KEY',
+import { z } from 'zod'
+
+const schema = z.object({
+	NODE_ENV: z.enum(['production', 'development', 'test'] as const),
```

Comment:
I'm concerned about the `enum` use here. Does that mean I can't do this:

```
if (process.env.NODE_ENV === 'development') {
}
```

With enums you normally have to reference some type for the equality check. Normally I would do this instead:

```suggestion
	NODE_ENV: z.union([z.literal('production'), z.literal('development'), z.literal('test')]),
```